packages/carbon/src/classes/components/FileUpload.ts

```diff
@@ -0,0 +1,37 @@
+import {
+	type APIFileUploadComponent,
+	BaseModalComponent
+} from "../../abstracts/BaseModalComponent.js"
+
+export abstract class FileUpload extends BaseModalComponent {
+	readonly type = 18 // ComponentType.FileUpload
```

Comment:
`18` -> `19` (though not part of `ComponentType` so type error when set to `19` atm)